src/Hl7.Fhir.Serialization.R4.Tests/RoundtripNewSerializers.cs

```diff
@@ -6,6 +6,8 @@ namespace Hl7.Fhir.Serialization.Tests
     [TestClass]
     public partial class RoundTripNewSerializers
     {
+        private readonly string _attachmentJson = "{\"size\":12}";
```

Comment:
It seems we now have a different serialization for long values for R4/R4B and R5? Scary.  That's the first time this happens in 10 years, truly scary.

Reply:
I mistunderstood this - in R4 the datatype was different, so the serialization changed too.

Reply:
See my general comment in this PR.